components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/index.jsp

```diff
@@ -220,34 +221,34 @@
                         }
 
                 %>
-                <tr id=<%=domainId%>>
+                <tr id=<%=Encode.forHtmlAttribute(domainId)%>>
                     <td style="width: 5%;margin-top:10px;">
                         <input type="checkbox" name="userStores"
-                               value="<%=domainId%>"
+                               value="<%=Encode.forHtmlAttribute(domainId)%>"
                                onclick="resetVars()"
                                class="chkBox"/>
                     </td>
                     <td style="width: 10%;margin-top:10px;">
-                        <a><%=domainId%>
+                        <a><%=Encode.forHtml(domainId)%>
                         </a>
                     </td>
                     <td style="width: 40%;margin-top:10px;">
-                        <a><%=className%>
+                        <a><%=Encode.forHtml(className)%>
                         </a>
                     </td>
                     <td style="width: 45%;margin-top:10px;">
                         <a title="<fmt:message key='edit.userstore'/>"
-                           onclick="edit('<%=domainId%>','<%=className%>');"
+                           onclick="edit('<%=Encode.forJavaScript(Encode.forHtml(domainId))%>','<%=Encode.forJavaScript(Encode.forHtml(className))%>');"
```

Comment:
Here you have encoded the attributes passed to the event. Thus, you can use Encode.forJavaScriptAttribute() instead of Encode.forJavaScript(). The edit function uses the passed attributes as query params in a URL. So it's wrong to encode them for html. You should use percent encoding instead with Encode.forUriComponent()